api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClient.java

```diff
@@ -16,6 +16,8 @@
   /** lists all notebook runtimes as the appengine SA, to be used only for admin operations */
   List<LeonardoListRuntimeResponse> listRuntimesByProjectAsService(String googleProject);
 
+  List<LeonardoListRuntimeResponse> listRuntimesByProject(String googleProject);
```

Comment:
I would clarify that this includes deleted runtimes, e.g. `listRuntimesByProjectWithDeleted()`; or else plumb the boolean through. Distinction is otherwise unclear between these two methods